# Think about how you could write a test that checks if a particular error 
# is raised. What kind of problems does that cause?

def test_exception_was_raised(func, args, message):